pkg/admission/initializers/initializer.go

```diff
@@ -139,3 +162,39 @@ func (i *shardExternalURLInitializer) Initialize(plugin admission.Interface) {
 		wants.SetShardExternalURL(i.shardExternalURL)
 	}
 }
+
+// NewKubeQuotaConfigurationInitializer returns an admission plugin initializer that injects quota.Configuration
+// into admission plugins.
+func NewKubeQuotaConfigurationInitializer(quotaConfiguration quota.Configuration) *kubeQuotaConfigurationInitializer {
+	return &kubeQuotaConfigurationInitializer{
+		quotaConfiguration: quotaConfiguration,
+	}
+}
+
+type kubeQuotaConfigurationInitializer struct {
+	quotaConfiguration quota.Configuration
+}
+
+func (i *kubeQuotaConfigurationInitializer) Initialize(plugin admission.Interface) {
+	if wants, ok := plugin.(initializer.WantsQuotaConfiguration); ok {
+		wants.SetQuotaConfiguration(i.quotaConfiguration)
+	}
+}
```

Comment:
this is preexisting `initializer.WantsQuotaConfiguration`

Reply:
Similar to the preceding comment, it didn't seem like the best approach to instantiate one of https://github.com/kcp-dev/kubernetes/blob/feature-logical-clusters-1.24/pkg/kubeapiserver/admission/initializer.go.

Reply:
That combines a RESTMapper with the QuotaConfiguration. Not ideal for us.

Also, this is constructed in https://github.com/kcp-dev/kubernetes/blob/feature-logical-clusters-1.24/pkg/genericcontrolplane/admission/config.go, which I don't think is used anywhere. But it constructs `quotainstall.NewQuotaConfigurationForAdmission()`, which includes the custom pod/pvc/service evaluators which we can't use right now. I create a different configuration in kcp's server.go.

Reply:
ack